centaur/src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_local.wdl

```diff
@@ -92,7 +92,7 @@ task modify_file_sfs {
     }
     runtime {
         # No docker, we need this to run against the same FS as Cromwell so it can modify the right file!
```

Comment:
Does this comment still apply? I don't understand what I meant before so I can't tell if it still makes sense if we're switching from Docker to GCR.

Reply:
Yes, I think it's still relevant. Those are still docker images, we are just switching container registry we use from Dockerhub to GCR and in this particular task we don't use any container since it seems to run locally

Reply:
Oh... I totally read this wrong. 🤦  I read it as, "No, docker..." as if we were trying to get docker to make a different decision, but it really means "this task doesn't need its own docker runtime" (or something like that).